app/views/routes/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_for([@route.user, @route]) do |f| %>
```

Comment:
`simple_form` and `slim-rails` would make this code easier to write, and much shorter.